src/bindings/c/tests/ov_property_test.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include "ov_test.hpp"
+
+TEST(ov_property, ov_properties_create_test) {
+    ov_properties_t* property = nullptr;
+    OV_ASSERT_OK(ov_properties_create(&property));
+
+    ov_properties_free(property);
+}
+
+TEST(ov_property, ov_properties_create_and_add) {
+    ov_properties_t* property = nullptr;
+    OV_ASSERT_OK(ov_properties_create(&property));
+
+    ov_property_key_e key = ov_property_key_e::PERFORMANCE_HINT;
+    ov_performance_mode_e mode = ov_performance_mode_e::THROUGHPUT;
+    ov_any_t value_1;
+    value_1.ptr = (void*)&mode;
+    value_1.size = 1;
+    value_1.type = ov_any_type_e::ENUM;
```

Comment:
BTW, is it possible to write with 1 line?
```c
ov_properties_add(property, key, { (void*)&mode, 1, ov_any_type_e::ENUM });
```
taking into account that `ov_properties_add` accepts last argument by value

Reply:
of course, it can be written into 1 line, it is up to C API user.